posit_team_contact_info.md

```diff
@@ -0,0 +1,41 @@
+# Posit Team Applications - Contact Information
+
+Original author(s): Kat Reid, Terry McLaughlin
+
+Last updated: January 2023
```

Comment:
As we're using version control, this isn't necessary?
```suggestion

```

Reply:
Agreed, I will remove from all guidance pages.